dhnx/graph.py

```diff
@@ -29,17 +29,17 @@ def thermal_network_to_nx_graph(thermal_network):
     """
     nx_graph = nx.MultiDiGraph()  # TODO: Check if this line can be removed.
 
-    edge_attr = list(thermal_network.components['edges'].columns)
```

Comment:
Knowledge about available_components is implicit here.

Reply:
Suggestion: Let thermal_network as it is, with its concrete components defined in component_attributes.csv. Nodes like consumers, producers are connected by pipes. However, when this function talks about graphs, talk about nodes and edges.